doc/nrf/protocols/matter/end_product/certification.rst

```diff
@@ -286,31 +286,27 @@ Nordic Semiconductor provides the following certification identifiers:
 
 You can visit the following pages to check the Bluetooth QDIDs and Thread CIDs valid for SoCs that support Matter applications:
 
-* `nRF5340 DK Compatibility Matrix`_
-* `nRF52840 DK Compatibility Matrix`_
+* `nRF5340 Compatibility Matrix <nRF5340 DK Compatibility Matrix_>`_
+* `nRF52840 Compatibility Matrix <nRF52840 DK Compatibility Matrix_>`_
+* `nRF54L15 Compatibility Matrix <nRF54L15 DK Compatibility Matrix_>`_
 
 Matter certification process variants
 *************************************
 
 The standard Matter certification process has several variants that you can use if you have already obtained the certification or you want to certify several products of the same family.
+There are separate processes for certification for different devices and recertification for devices that are already certified.
 
-Rapid Recertification program
-=============================
-
-.. note::
-   The Rapid Recertification program is currently experimental.
-   You can read more about it in a dedicated document in the `Matter Resource Kit`_.
-
-The Rapid Recertification program lets you perform the product testing using the services of a qualified CSA Member, even your own organization, instead of an ATL.
-This allows for a potentially more streamlined recertification process when you want to certify
-The first certification must still be done at an ATL.
+Portfolio Certification program
+===============================
 
-.. figure:: images/matter_device_certification_process_rr.svg
-   :alt: Matter's Rapid Recertification program overview
+This certification program applies to both Zigbee and Matter products.
+It simplifies the certification process by allowing manufacturers to certify multiple Portfolio Member Products within a single application, using a Parent Product as the basis.
+This program offers greater flexibility compared to the existing Certification by Similarity and Product Family Certification Programs.
```

Comment:
@doublemis1 could we give an example - why flexibility is greater?

Reply:
Basic difference is that the product family certification is for the same device and the child device can be only the same device with different housing, device language, power plug etc. (form the matter stand point it is the same device), while the portfolio accepts to have a devices which may be different on matter level e.g. Parent certificate is a master set of the device and each other device with some features removed can leverage the product portfolio certification

Reply:
I'm afraid I cannot make anything sensible out of this. Could you suggest a simple text.

Reply:
Gave AI a chance. @LuDuda, please check.